containers/README.md

```diff
@@ -18,13 +18,16 @@ To run containerized Kraken as a Kubernetes/OpenShift Deployment, follow these s
     - In Kubernetes, use `kubectl config set-context --current --namespace=<namespace>`
     - In OpenShift, use `oc project <namespace>`
-4. Create a ConfigMap named kube-config using `kubectl create configmap kube-config --from-file=<path_to_kubeconfig>`
-5. Create a ConfigMap named kraken-config using `kubectl create configmap kraken-config --from-file=<path_to_kraken_config>`
-6. Create a ConfigMap named scenarios-config using `kubectl create configmap scenarios-config --from-file=<path_to_scenarios_folder>`
-7. Create a ConfigMap named scenarios-openshift-config using `kubectl create configmap scenarios-openshift-config --from-file=<path_to_scenarios_openshift_folder>`
-8. Create a ConfigMap named scenarios-kube-config using `kubectl create configmap scenarios-kube-config --from-file=<path_to_scenarios_kube_folder>`
-9. Create a service account to run the kraken pod `kubectl create serviceaccount useroot`.
-10. In Openshift, add privileges to service account and execute `oc adm policy add-scc-to-user privileged -z useroot`.
-11. Create a Job using `kubectl apply -f kraken.yml` and monitor the status using `oc get jobs` and `oc get pods`.
-
-NOTE: It is not recommended to run Kraken internal to the cluster as the pod which is running Kraken might get disrupted.
+4. Create a ConfigMap named kube-config using `kubectl create configmap kube-config --from-file=<path_to_kubeconfig>` *(eg. ~/.kube/config)*\
+***NOTE:** your kubeconfig might contain several cluster contexts and credentials so be sure, before creating the ConfigMap, to keep **only** the credentials related to the destination cluster. Please refer to the [Kubernetes documentation](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) for more details*
+1. Create a ConfigMap named kraken-config using `kubectl create configmap kraken-config --from-file=<path_to_kraken>/config`
+2. Create a ConfigMap named scenarios-config using `kubectl create configmap scenarios-config --from-file=<path_to_kraken>/scenarios` 
+3. Create a ConfigMap named scenarios-openshift-config using `kubectl create configmap scenarios-openshift-config --from-file=<path_to_kraken>/scenarios/openshift`
+4. Create a ConfigMap named scenarios-kube-config using `kubectl create configmap scenarios-kube-config --from-file=<path_to_kraken>/scenarios/kube` \
+***NOTE**: both the scenarios ConfigMaps are needed regardless you're running kraken in Kubernetes or OpenShift*
+9.  Create a service account to run the kraken pod `kubectl create serviceaccount useroot`.
+10. In Openshift, add privileges to service account and execute `oc adm policy add-scc-to-user privileged -z useroot`.\
+***NOTE:** to add privileges to the service account you must be logged in the cluster with an highly privileged account (ideally kubeadmin)*
+11. Create a Job using `kubectl apply -f <path_to_kraken>/containers/kraken.yml` and monitor the status using `oc get jobs` and `oc get pods`.
```

Comment:
Maybe add the command to monitor the status of the job since we know the name of the job - https://github.com/redhat-chaos/krkn/blob/main/containers/kraken.yml#L5. Thoughts?